README.md

```diff
@@ -775,11 +778,127 @@ routes => [
 #####`type`
 *Optional* The type of VPN gateway. The only currently supported value --- and the default --- is 'ipsec.1'.
 
-#### Type: route53*
+#### Type: rds_db_parameter_group
+
+Note that this type is currently only available to list via puppet
+resource and cannot be created by Puppet.
+
+#####`name`
+*Required* The name of the parameter group.
+
+#####`region`
+*Required* The region in which to launch the parameter group. For valid values, see [AWS Regions](http://docs.aws.amazon.com/general/latest/gr/rande.html#ec2_region).
+
+#####`description`
+A description of the parameter group.
```

Comment:
All of these new parameters should say what the acceptable or valid values are. Something like "Accepts a string." or "Valid options: true or false." should be fine (assuming that's correct, of course!)